pkg/plugins/discovery/k8s/controllers/pod_converter.go

```diff
@@ -42,8 +44,10 @@ func (p *PodConverter) DataplaneFor(pod *kube_core.Pod, services []*kube_core.Se
 		Networking: &mesh_proto.Dataplane_Networking{},
 	}
 	if injector_metadata.HasTransparentProxyingEnabled(pod) {
+		services := pod.GetAnnotations()[injector_metadata.KumaDirectAccess]
```

Comment:
Shall we have some sanity checks for the services? What if there is some garbage that string.Split can't handle?

Reply:
How would this sanity check look like? If someone put garbage here like `!@#$`, we will just see if there is such service and ignore if not.